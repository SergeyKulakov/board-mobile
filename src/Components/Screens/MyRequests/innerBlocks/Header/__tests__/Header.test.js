import React from 'react'
import { shallow } from 'enzyme'
import { expect as chaiExpect } from 'chai'

import 'jest-styled-components'

import { Header as Component } from '../index'

describe('MyRequests.Header', () => {
  const defaultProps = {
    activeTabId: 0,
    onBackClick: () => null,
    onClickTab: () => null,
    onHamburgerClick: () => null,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
