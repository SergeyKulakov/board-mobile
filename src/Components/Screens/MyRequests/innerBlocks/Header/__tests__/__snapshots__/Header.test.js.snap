// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyRequests.Header render snapshot 1`] = `
<Component>
  <Styled(View)>
    <Styled(View)>
      <Styled(View)>
        <Styled(BackIcon)
          onClick={[Function]}
        />
        <Component
          onClick={[Function]}
        />
      </Styled(View)>
      <Styled(View)>
        <Styled(ScreenTitle)>
          My Requests
        </Styled(ScreenTitle)>
      </Styled(View)>
      <Styled(View) />
    </Styled(View)>
    <Styled(View)>
      <TabNavigationElement
        isActive={true}
        onClick={[Function]}
      >
        Booking Requests
      </TabNavigationElement>
      <TabNavigationElement
        isActive={false}
        onClick={[Function]}
      >
        Hiring Requests
      </TabNavigationElement>
    </Styled(View)>
  </Styled(View)>
</Component>
`;
